src/compiler/output-targets/test/custom-elements-types.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { path } from '@stencil/core/compiler';
+import { mockConfig, mockStencilSystem, mockBuildCtx, mockCompilerCtx, mockModule } from '@stencil/core/testing';
+import type * as d from '../../../declarations';
+import * as outputCustomElementsMod from '../dist-custom-elements';
+import { stubComponentCompilerMeta } from '../../types/tests/ComponentCompilerMeta.stub';
+import { generateCustomElementsTypes } from '../dist-custom-elements/custom-elements-types';
+
+const setup = () => {
+  const sys = mockStencilSystem();
+  const config: d.Config = mockConfig(sys);
+  const compilerCtx = mockCompilerCtx(config);
+  const buildCtx = mockBuildCtx(config, compilerCtx);
+  const root = config.rootDir;
+  config.configPath = '/testing-path';
+  config.srcDir = '/src';
+  config.buildAppCore = true;
+  config.rootDir = path.join(root, 'User', 'testing', '/');
+  config.namespace = 'TestApp';
+  config.buildEs5 = true;
+  config.globalScript = path.join(root, 'User', 'testing', 'src', 'global.ts');
+  config.outputTargets = [{ type: 'dist-custom-elements', dir: 'my-best-dir' }];
```

Comment:
Does it make sense for us to use the `DIST_CUSTOM_ELEMENTS` constant here for the type of our output target?
```suggestion
  config.outputTargets = [{ type: DIST_CUSTOM_ELEMENTS, dir: 'my-best-dir' }];
```

(we'll have to import it if we go this route)

Reply:
🙊 so I did!